src/DocumentFormat.OpenXml/Packaging/Model3DReferenceRelationshipPart.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+using DocumentFormat.OpenXml.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO.Packaging;
+
+namespace DocumentFormat.OpenXml.Packaging
+{
+    /// <summary>
+    /// Defines the Model3DReferenceRelationshipPart
+    /// </summary>
+    public partial class Model3DReferenceRelationshipPart : OpenXmlPart, IFixedContentTypePart
+    {
+        /// <summary>
+        /// Determines if the content type agrees with this part's constraints.
+        /// </summary>
+        /// <param name="metroPart"></param>
+        /// <returns></returns>
+        private protected override bool IsValidContentType(PackagePart metroPart)
+        {
+            if (IsContentTypeFixed && metroPart.ContentType != ContentType)
+            {
+                return metroPart.ContentType == "model/gltf.binary";
+            }
+
+            return true;
```

Comment:
what about it's default type? I'd imagine you'd want to call `base.IsValidContentType(metroPart)` at some point

Reply:
I think you're right. I initially wrote it like the base so it wouldn't need to call the base but really we should use a pattern of calling the base first, IMO.

Reply:
I changed it to call the base first then do the special case.

Reply:
As-is, the base method isn't doing anything....